internal/cri/server/podsandbox/sandbox_run_linux_test.go

```diff
@@ -142,7 +143,12 @@ func TestLinuxSandboxContainerSpec(t *testing.T) {
 					Type: runtimespec.IPCNamespace,
 				})
 				assert.Contains(t, spec.Linux.Sysctl["net.ipv4.ip_unprivileged_port_start"], "0")
-				assert.Contains(t, spec.Linux.Sysctl["net.ipv4.ping_group_range"], "0 2147483647")
+				if userns.RunningInUserNS() {
+					assert.Equal(t, spec.Linux.Sysctl["net.ipv4.ping_group_range"], "")
```

Comment:
Why empty string? The sysctl still exists in the userns, no?

Reply:
I'm not sure either, that's what the code currently does.

Reply:
Probably better to just skip the check when in userns

Reply:
sure thing, will do